src/env_loader.zig

```diff
@@ -909,58 +534,251 @@ pub const Loader = struct {
     }
 };
 
-pub const Parser = struct {
-    pub fn parse(
-        source: *const logger.Source,
+const Parser = struct {
+    pos: usize = 0,
+    src: string,
+
+    const whitespace_chars = "\t\x0B\x0C \xA0\n\r";
+
+    fn skipLine(this: *Parser) void {
+        var i = this.pos;
+        while (i < this.src.len) : (i += 1) {
+            switch (this.src[i]) {
+                '\n' => {
+                    this.pos = i + 1;
+                    return;
+                },
+                '\r' => {
+                    i += 1;
+                    this.pos = if (i < this.src.len and this.src[i] == '\n') i + 1 else i;
+                    return;
+                },
+                else => {},
+            }
+        }
+        this.pos = i;
+    }
+
+    fn skipWhitespaces(this: *Parser) void {
+        var i = this.pos;
```

Comment:
Do we need to care about non-ascii whitespace characters?

Reply:
Do you mean `\u{2028}` etc.?

I don't believe anybody relies on that, but we can always add support for it if a use case in the wild pops up.